x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/constants.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ASCENDING = (fieldName: string) =>
+  i18n.translate('xpack.enterpriseSearch.appSearch.documents.search.sortBy.option.ascending', {
```

Comment:
Is this only used in the an EUI Select dropdown? if so we might want to use the `DropDownOptionLabel` suffix for this per https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types:

```suggestion
  i18n.translate('xpack.enterpriseSearch.appSearch.documents.search.sortBy.option.ascendingDropDownOptionLabel', {
```

Also good call on making this a fn!

Reply:
https://github.com/elastic/kibana/pull/88764/commits/88143b18dfb85cc319ab52f8b00b6acbd6f2c472